filter/grok/filtergrok.go

```diff
@@ -64,16 +64,24 @@ func InitHandler(ctx context.Context, raw *config.ConfigRaw) (config.TypeFilterC
 // Event the main filter event
 func (f *FilterConfig) Event(ctx context.Context, event logevent.LogEvent) logevent.LogEvent {
 	message := event.GetString(f.Source)
-	values, err := f.grk.Parse(f.Match, message)
+	var err error
+	for _, thisMatch := range f.Match {
+		values, err := f.grk.Parse(thisMatch, message)
```

Comment:
1. err may be shadow, so line 80 will not cache any error
2. err may be overwrite at the next for loop, so some error may be ignore

Reply:
1. I'm not sure whether the lexer would figure out that err already exist or if it needs values to be defined in a line beforehand. Stupid lexical ambiguity ...
2. The overwriting thing was intentional -- the idea is that we skip to the next pattern if we don't find one, otherwise we'd be writing a series of errors every time the first pattern wasn't matched but the fifth or sixth one was.

Reply:
OK
But github think this PR was merged.
You should create a new PR for merging to master branch.